### ISSUE TITLE
incorrect link for system color in aria-disable page

### ISSUE THREAD

**Issue by kosl90** [mdn/content#27450]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

### What information was incorrect, unhelpful, or incomplete?

the link for 'system color' is incorrect

### What did you expect to see?

the correct link should be https://developer.mozilla.org/en-US/docs/Web/CSS/system-color#syntax

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/attributes/aria-disabled`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/attributes/aria-disabled/index.md
* Last commit: https://github.com/mdn/content/commit/5b5fadb2cb1d21c43abacffe2f58ab4ec76ca543
* Document last modified: 2023-05-31T13:27:00.000Z

</details>
